src/reanimated2/js-reanimated/JSReanimated.ts

```diff
@@ -158,4 +170,32 @@ export default class JSReanimated extends NativeReanimated {
         return 'AbsoluteOrientationSensor';
     }
   }
+
+  detectPlatform() {
+    const userAgent = navigator.userAgent || navigator.vendor || window.opera;
+    if (userAgent === undefined) {
+      return;
```

Comment:
Shouldn't we assign `Platform.UNKNOWN` here so that we won't call `detectPlatform` again?